library/core/src/num/f16.rs

```diff
@@ -435,6 +435,7 @@ impl f16 {
         // WASM, see llvm/llvm-project#96437). These are platforms bugs, and Rust will misbehave on
         // such platforms, but we can at least try to make things seem as sane as possible by being
         // careful here.
+        // Cc https://github.com/rust-lang/rust/issues/114479
```

Comment:
...carbon copy?

shouldn't this be ["quod vide" (qv.)](https://en.wikipedia.org/wiki/List_of_Latin_phrases_(full)#quod_vide) or ["conferatur" (cf.)](https://en.wikipedia.org/wiki/Cf.)? or just "see also"?
```suggestion
        // see also https://github.com/rust-lang/rust/issues/114479
```